catalog/ui/search-ui/standard/src/main/webapp/js/view/preferences/PreferencesModal.view.js

```diff
@@ -320,9 +320,10 @@ define([
             ui: {
                 tbody: 'tbody'
             },
+            viewComparator: 'label',
             initialize: function() {
                 this.collection.each(function(model) {
```

Comment:
It doesn't flow the change in the model up, which is why I looped over the models.